app/helpers/news_helper.rb

```diff
@@ -0,0 +1,21 @@
+module NewsHelper
+  def news_url(news)
+    if news.present? && news.url.present?
+      content_tag(:span, class: :link) do
+        link_to(news.url, target: :blank) do
+          fa_icon('link') + " " + news.url
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.